contrib/dev/Vagrantfile

```diff
@@ -16,9 +32,16 @@ $skydive_extra_config = {
   "storage.elasticsearch.host" => "127.0.0.1:9200"
 }
 
+$skydive_git_clone = <<SCRIPT
+mkdir -p #{SKYDIVE_GUEST}
+cd #{SKYDIVE_GUEST}
+git clone https://#{SKYDIVE_GITHUB} .
+ssh-keygen -f id_rsa -t rsa -N ''
```

Comment:
I think this is out of scope of the "devmode". If the idea is to use the VM as a primary dev environment - adding the generated key to Github -, this can be done manually once with "vagrant ssh ssh-keygen -f id_rsa -t rsa -N"'

Reply:
ok, removing